File: FTGCircularTuner/CircularTunerDemo/CircularTunerDemo/ViewController.m
//
//  ViewController.m
//  CircularTunerDemo
//
//  Created by Khoa Pham on 4/11/15.
//  Copyright (c) 2015 Fantageek. All rights reserved.
//

#import "ViewController.h"
#import "FTGCircularTuner.h"

@interface ViewController ()

@property (nonatomic, strong) FTGCircularTuner *tuner;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    FTGCircularTuner *tuner = [[FTGCircularTuner alloc] init];
    tuner.incompleteColor = [UIColor grayColor];
    tuner.image = [UIImage imageNamed:@"volume"];

    [tuner addTarget:self action:@selector(tunerDidBeginInteraction:) forControlEvents:UIControlEventEditingDidBegin];
    [tuner addTarget:self action:@selector(tunerDidEndInteraction:) forControlEvents:UIControlEventEditingDidEnd];
    [tuner addTarget:self action:@selector(tunerProgressChanged:) forControlEvents:UIControlEventValueChanged];

    self.tuner = tuner;
    [self.view addSubview:tuner];
}

- (void)viewDidLayoutSubviews {
    [super viewDidLayoutSubviews];

    self.tuner.frame = CGRectMake(0, 0, 200, 250);
    self.tuner.center = self.view.center;
}

#pragma mark - Event
- (void)tunerDidBeginInteraction:(id)sender {
    NSLog(@"begin");
}

- (void)tunerDidEndInteraction:(id)sender {
    NSLog(@"end");
}

- (void)tunerProgressChanged:(FTGCircularTuner *)tuner {
    NSLog(@"progress %f", tuner.progress);
}



@end

File: StripeForm/Example/StripeForm/ViewController.swift
//
//  ViewController.swift
//  StripeForm
//
//  Created by Khoa Pham on 12/28/2015.
//  Copyright (c) 2015 Khoa Pham. All rights reserved.
//

import UIKit
import StripeForm

class ViewController: UIViewController {
    var form: StripeForm!

    override func viewDidLoad() {
        super.viewDidLoad()

        setupForm()
    }

    func setupForm() {
        form = StripeForm.makeForm()
        form.translatesAutoresizingMaskIntoConstraints = false
        view.addSubview(form)

        form.leadingAnchor.constraintEqualToAnchor(view.leadingAnchor, constant: 20).active = true
        form.trailingAnchor.constraintEqualToAnchor(view.trailingAnchor, constant: -20).active = true
        form.topAnchor.constraintEqualToAnchor(view.topAnchor, constant: 200).active = true
    }
}

